forecasttools/daily_to_epiweekly.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+
+import polars as pl
+
+
+# NOTE: this needs to change to ACTUAL epiweeks
+def calculate_epidate(date):
+    iso_calendar = date.isocalendar()
+    return iso_calendar[1], iso_calendar[0]
+
+
+def daily_to_epiweekly(
+    forecast_df: pl.DataFrame,
+    value_col: str = "value",
+    date_col: str = "date",
+    id_cols: list[str] = [".draw"],
+    weekly_value_name: str = "weekly_value",
+    strict: bool = False,
```

Comment:
I can read the code to know what this does, but wondering if it could be documented in a docstring or be given a more clear name

Reply:
DHM has very informative docstrings. I will anchor on these for the part. Variable naming, comment informativeness, and docstrings are something I am working on today.